bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -29,13 +32,54 @@
  * A view displaying the HTML report generated by PIT.
  */
 public class PitView extends ViewPart implements SummaryView {
+    public final static String VIEW_ID = "org.pitest.pitclipse.ui.view.PitView";
```

Comment:
`static` should be written before `final`: `public static final`

This is a naming convention. You can install the Checkstyle plug-in to catch such warnings.

Reply:
Will fix.

I installed Checkstyle, but it shows more than 400 warnings for files I never touched, and after trying unsuccessfully to solve the problem, I decided to uninstall it.